esm/dom/parser.js

```diff
@@ -4,24 +4,33 @@ import {parseFromString} from '../shared/parse-from-string.js';
 import {HTMLDocument} from '../html/document.js';
 import {SVGDocument} from '../svg/document.js';
 import {XMLDocument} from '../xml/document.js';
+import {MIME} from '../shared/symbols.js';
 
 /**
  * @implements globalThis.DOMParser
  */
 export class DOMParser {
 
   /** @typedef {{ "text/html": HTMLDocument, "image/svg+xml": SVGDocument, "text/xml": XMLDocument }} MimeToDoc */
+  /** @typedef {{ [x: symbol]: unknown, [MIME] : string }} Config */
   /**
    * @template {keyof MimeToDoc} MIME
    * @param {string} markupLanguage
-   * @param {MIME} mimeType
+   * @param {Config | string} config
    * @returns {MimeToDoc[MIME]}
    */
-  parseFromString(markupLanguage, mimeType) {
+  parseFromString(markupLanguage, config) {
+    const [mimeType, domOptions] = ((c) => {
+      if (typeof c === 'object') {
+        const {[MIME]: mime, ...rest} = c;
+        return [mime || 'text/html', rest]
+      }
+      return [c, {}]
+    })(config);
```

Comment:
The second argument can now look like: 
- "text/html"
- {[WINDOW]: {}}
- {mime: "text/html", [WINDOW]: {}}
- {[MIME]: "text/html", [WINDOW]: {}}

Reply:
what's the reason for `MIME` symbol, if I might ask?

Reply:
The symbols here are not needed. The only benefit now is that the `config` object has only `Symbol` keys. It would be fine to have the string "mime" and the symbol `WINDOW` for keys...

Actually, it would even be okay to have the strings "mime" and "window" as keys. At the `parseFromString` step, there is currently no risk of namespace conflict. The `WINDOW` symbol of "linkedomWindowProxy" only must be used internally [here](https://github.com/WebReflection/linkedom/pull/104/files#diff-c5f236b00784b2abd39c39c877ff6115c540f6cf142112bba3ad49ae7bfda28cR125).